authz/sdk_end2end_test.go

```diff
@@ -379,6 +379,30 @@ func (s) TestSDKFileWatcherEnd2End(t *testing.T) {
 	}
 }
 
+func verifyValidReload(ctx context.Context, tsc pb.TestServiceClient, wantCode codes.Code, wantErr string) (lastStatus *status.Status) {
+	for numRetries := 0; numRetries <= 20; numRetries++ {
+		_, err := tsc.UnaryCall(ctx, &pb.SimpleRequest{})
+		if lastStatus = status.Convert(err); lastStatus.Code() == wantCode && lastStatus.Message() == wantErr {
+			return nil
+		}
+		time.Sleep(20 * time.Millisecond)
+		numRetries++
+	}
+	return lastStatus
+}
+
+func verifySkipReload(ctx context.Context, tsc pb.TestServiceClient, wantCode codes.Code, wantErr string) (lastStatus *status.Status) {
+	for numRetries := 0; numRetries <= 20; numRetries++ {
+		_, err := tsc.UnaryCall(ctx, &pb.SimpleRequest{})
+		if lastStatus := status.Convert(err); lastStatus.Code() != wantCode || lastStatus.Message() != wantErr {
```

Comment:
This seems wrong.  This isn't waiting for a reload to happen, this is waiting to be sure a change did not happen after there was a reload.  For that test, we can sleep for (interval+epsilon) and just check once.  Set the interval back down to something very low for these test cases so we don't have to wait for long.

Reply:
Sure. I updated the tests. PTAL.